builtin/logical/pki/path_fetch.go

```diff
@@ -96,6 +110,37 @@ func (b *backend) pathFetchCertList(req *logical.Request, data *framework.FieldD
 	return logical.ListResponse(entries), nil
 }
 
+func (b *backend) pathFetchReadCAChain(req *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	caInfo, err := fetchCAInfo(req)
+	switch err.(type) {
+	case errutil.UserError:
+		return nil,
+			errutil.UserError{Err: fmt.Sprintf("Could not fetch the CA certificate: %s", err)}
+	case errutil.InternalError:
+		return nil,
+			errutil.InternalError{Err: fmt.Sprintf("Error fetching CA certificate: %s", err)}
+	}
+
+	caChain := caInfo.GetCAChain()
+	certs := []byte{}
+	for _, ca := range caChain {
+		block := pem.Block{
+			Type:  "CERTIFICATE",
+			Bytes: ca.Bytes,
+		}
+		certs = append(certs, pem.EncodeToMemory(&block)...)
+	}
+
+	response := &logical.Response{
```

Comment:
This tweaks me a little in a consistency sense.

`ca` returns a Vault response; `ca/pem` returns a raw response.
`crl` returns a Vault response; `crl/pem` returns a raw response.

It *feels* like `ca/chain` should return a Vault response and `ca/chain/pem` should return a raw response. Although in that scenario it's probably better to just use `ca_chain` and `ca_chain/pem`.

The reason for having both is so that the Vault CLI (and API clients, generally) can do a read against one of the endpoints and understand how to parse it.